samba/build/build_deb

```diff
@@ -34,8 +34,13 @@ fi
 sudo cp $WORKSPACE/shaman.list /etc/apt/sources.list.d/
 
 ## Install any setup-time deps
-# Make sure we use the latest version of libcephfs-dev
-sudo apt-get remove -y libcephfs-dev || true
+# Make sure we use the latest ceph versions, remove any old bits
+xargs sudo apt-get remove -y <<< "
+libcephfs-dev
+libcephfs1
+libcephfs2
+librados2
+"
```

Comment:
we've seen issues with librdb1 as well (unsure if we now have a librbd2?)

https://github.com/ceph/ceph-deploy/blob/master/ceph_deploy/install.py#L307-L308